src/App.css

```diff
@@ -1,4 +1,317 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&display=swap');
+
 body {
   padding: 0;
   text-align: center;
+  width: 100%;
+  margin: auto;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+#root {
+  width: 100%;
+  position: absolute;
+  top: 0;
+  background-color: #fafafa;
+}
+
+.navBar {
+  display: flex;
+  margin: auto;
+  padding: 1rem 6rem;
+  justify-content: space-between;
+  background-color: white;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+.Bookstore-CMS {
+  width: auto;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.half {
+  flex: 1;
+}
+
+.menuItems {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat, serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+  list-style: none;
+  display: flex;
+}
+
+.links {
+  color: #121212;
+}
+
+.menuItems li {
+  padding: 0 1rem;
+  font-size: 0.8rem;
+  text-transform: uppercase;
+}
+
+.leftMenu {
+  display: flex;
+  align-items: center;
+}
+
+.Book-Panel {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 75rem;
+  height: 10.625rem;
+  margin: 1rem auto;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #e8e8e8;
+}
```

Comment:
- [ ] Kindly align the book panel with the title to be similar to the Zeplin design.

| Zeplin | Your design |
|---|---|
| ![image](https://github.com/zamcham/Bookstore/assets/58642949/942fe4b9-d1f2-4fe1-9a10-a692bc2989bd) | ![image](https://github.com/zamcham/Bookstore/assets/58642949/d11dbb83-6f73-40b2-a05c-e145c6c153a5) |